.gitlab-ci.yml

```diff
@@ -114,19 +114,26 @@ Build and Install:
     expire_in: 1 day
 
 Load REST Libraries:
-  stage: install
+  stage: simpleRun
   script:
     - echo ${CI_PROJECT_DIR}/install/thisREST.sh
     - cat ${CI_PROJECT_DIR}/install/thisREST.sh
     - source ${CI_PROJECT_DIR}/install/thisREST.sh
     - restRoot -b -q
 
 List REST Macros:
-  stage: install
+  stage: simpleRun
   script:
     - source ${CI_PROJECT_DIR}/install/thisREST.sh
     - restManager ListMacros
 
+Open v2.2 Files:
```

Comment:
We are in the process of migrating to GitHub Actions so probably we should not add new steps into the GitLab pipeline, instead we should add them into the GitHub.

Reply:
I am not familar with GitHub Actions, I need to study a little.

Reply:
I have migrated the file opening tests to GitHub Actions